### PR TITLE
Add BaseIndexableTestCase + reorg utils

### PR DESCRIPTION
**Pull request by mparent61**
BaseIndexableTestCase can be shared among all DJES consumers

Had to re-org existing "util.py" to fit with more flexible "utils/" module/folder, and confirmed no external projects will be affected (util.py currently only used internally)

@MichaelButkovic @benghaziboy

### CODE REVIEW

**Comment by benghaziboy** [comment]
LGTM

File: djes/utils/test.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+from djes.management.commands.sync_es import get_indexes, sync_index
+
+
+class BaseIndexableTestCase(TestCase):
+    """A TestCase which handles setup and teardown of elasticsearch indexes."""
+
+    elasticsearchLogger = logging.getLogger('elasticsearch')
+
+    def setUp(self):
+        """ If you're reading this. I am gone and dead, presumably.
```
Comment (csinchok) [inline djes/utils/test.py]: :heart:

Comment (benghaziboy) [inline djes/utils/test.py]: can we delete this

Comment (csinchok) [inline djes/utils/test.py]: LGTM

Comment (benghaziboy) [inline djes/utils/test.py]: doesn't matter what it looks like to you. leave

Comment (mparent61) [inline djes/utils/test.py]: @clowe is just upset because I'm getting credit for his "dead and gone" comment

**Pull request merged**